doc/content/sqa/theory_manual.md

```diff
@@ -0,0 +1,11 @@
+# MALAMUTE Theory Manual
+
+Background on the basis of MALAMUTE theory can be found throughout the standard user documentation.
+Here, we highlight several locations of interest for MALAMUTE theory:
+
+- Documentation for all MALAMUTE input file objects, and how they can be used in the creation of a
+  multiphysics simulation, is contained in the [syntax/index.md] page.
+- MALAMUTE verification and validation (V&V) cases, along with associated documentation, can be found
+  in the [malamute_vvr.md].
+- Documentation related to demonstration cases (which include the V&V cases mentioned above) can be
+  found on the [demonstration/index.md] page.
```

Comment:
Is the addition of this page intended to be more in line with SQA documentation? If yes, what is the associated SQA guidance/ requirement that we can point to for this change?

Reply:
This better aligns us with PLN-4005, the SQA Plan for MOOSE and MOOSE-based applications. To get a little more specific, this also aligns with Corrective Action `CA 2023-1161` as presented in the INL LabWay system. Considering our V&V and demonstration cases contain theory descriptions, it seemed appropriate to create a landing page directing the user/developer to these locations, rather than just saying "NONE" here.